schema/table.go

```diff
@@ -24,6 +25,13 @@ const (
 var (
 	baseModelType      = reflect.TypeFor[BaseModel]()
 	tableNameInflector = inflection.Plural
+
+	// https://github.com/uptrace/bun/issues/1095
+	// < v1.2, all fields follow the order corresponding to the struct
+	// >= v1.2, < v1.2.8, fields of nested structs have been moved to the end.
+	// >= v1.2.8, The default behavior remains the same as initially,
+	// but we allow users to control this behavior through this parameter.
+	DisableSortFieldsByStruct = false
```

Comment:
I'd prefer that we don't introduce this option and instead always preserve the order of struct fields.